src/components/Patient/SampleDetails.tsx

```diff
@@ -305,169 +308,224 @@ export const SampleDetails = ({ id }: DetailRoute) => {
+              </div>
+              <div className="font-medium">
+                {sampleDetails?.date_of_result
+                  ? formatDateTime(sampleDetails.date_of_result)
+                  : "-"}
+              </div>
+            </div>
+          </div>
+
           {sampleDetails?.doctor_name && (
-            <div className="capitalize md:col-span-2">
-              <span className="font-semibold leading-relaxed">
-                {t("doctors_name")}:{" "}
-              </span>
-              <span id="doctor_name">
+            <div className="space-y-1">
+              <div className="text-sm text-muted-foreground">
+                {t("Doctor's Name")}:
+              </div>
+              <div id="doctor_name" className="font-medium">
                 {startCase(camelCase(sampleDetails.doctor_name))}
-              </span>
+              </div>
             </div>
           )}
-          {sampleDetails?.diagnosis && (
-            <div className="md:col-span-2">
-              <span className="font-semibold leading-relaxed">
-                {t("diagnosis")}:{" "}
-              </span>
-              <span id="diagnosis">{sampleDetails.diagnosis}</span>
+          <div className="border-t border-gray-300 my-4"></div>
+          <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-4">
+            {sampleDetails?.fast_track && (
+              <div
+                id="fast_track_reason"
+                className="flex justify-between items-center gap-2 p-3 bg-gray-100 rounded-lg"
+              >
+                <div className="font-medium text-sm">
+                  {t("fast_track_testing_reason")}:{" "}
+                </div>
+                <Badge variant={"secondary"}>{sampleDetails.fast_track}</Badge>
+              </div>
+            )}
+            {sampleDetails?.diagnosis && (
+              <div className="flex justify-between items-center gap-2 p-3 bg-gray-100 rounded-lg">
+                <div className="font-medium text-sm">{t("diagnosis")}: </div>
+                <Badge id="diagnosis" variant={"secondary"}>
+                  {" "}
+                  {sampleDetails.diagnosis}
+                </Badge>
+              </div>
+            )}
+            {sampleDetails?.diff_diagnosis && (
+              <div className="flex justify-between items-center gap-2 p-3 bg-gray-100 rounded-lg">
+                <div className="font-medium text-sm">
+                  {t("differential_diagnosis")}:{" "}
+                </div>
+                <Badge id="diff_diagnosis" variant={"secondary"}>
+                  {" "}
+                  {sampleDetails?.diff_diagnosis}
+                </Badge>
+              </div>
+            )}
+            {sampleDetails?.etiology_identified && (
+              <div className="flex justify-between items-center gap-2 p-3 bg-gray-100 rounded-lg">
+                <div className="font-medium text-sm">
+                  {t("Etiology identified")}:{" "}
```

Comment:
Compare the translations; they are based on key value pair, not the application directly translating the terms.

This should be etiology_identified for example.